crmsh/utils.py

```diff
@@ -1221,31 +1221,22 @@ def edit_file(fname):
     return ext_cmd_nosudo("%s %s" % (config.core.editor, fname))
 
 
-def edit_file_ext(fname, template=''):
+@contextmanager
+def edit_file_ext(fname):
```

Comment:
The interface looks strange. `edit_file_ext` does not finished the job of editing a file. How about `def edit_file_ext(fname: str, validator: typing.Callable[typing.IO, bool])`